File: runtime/python/object.py
import os

n = 1024 * 1024 * 10
a = []

print ('pid = %d; press a key to start'%(os.getpid()))
input()

for i in range(n):
    o = object()
    a.append(o)

print ('allocation is done. I will enter spinning. Find me with top')
while True:
    pass

File: runtime/go/object.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

type Object struct {
}

func main() {
	fmt.Printf("pid = %d; press a key to start\n", os.Getpid())
	r := bufio.NewReader(os.Stdin)
	r.ReadString('\n')
	
	n := 1024 * 1024 * 10
	a := []Object{}
	for i:= 0; i < n; i++ {
		o := Object{}
		a = append(a, o)
	}

	fmt.Println("allocation is done. I will enter spinning. Find me with top")
	for {}
}
